didntreally.CodeReviews.Console.Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Declare variables and then initialize to zero.
+            string num1 = "";
+            string num2 = "";
+            int usageNum = 0;
+            List<double> calculations = new List<double>();
+
+
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Welcome to the console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
+            bool runCalculator = true;
+
+            while(runCalculator)
+            {
+                //Ask the user to enter a number
+                
+                Console.WriteLine("Type a number and then press Enter! ");
+                num1 = Console.ReadLine();
+                double numInput1 = 0;
+                while (!double.TryParse(num1, out numInput1))
+                {
+                    Console.WriteLine("This is not a valid number, please enter a number! ");
+                    num1 = Console.ReadLine();
+                }
+
+
+
+
+                //Ask User to type in second numebr
+                Console.WriteLine("Type a second number, and then press Enter! ");
+                num2 = Console.ReadLine();
+                double numInput2 = 0;
+                while (!double.TryParse(num2, out numInput2))
+                {
+                    Console.WriteLine("This is not a valid number, please enter a number! ");
+                    num2 = Console.ReadLine();
+                }
+
+
+                //Ask the user to choose an option
+                Console.WriteLine("Choose an option from the list and type in the alphabet ");
+                Console.WriteLine("\ta - Add");
+                Console.WriteLine("\ts - Subtract");
+                Console.WriteLine("\tm - Multiply");
+                Console.WriteLine("\td - divide");
+                Console.WriteLine("What is your option: ");
+                string userInput = Console.ReadLine().ToUpper();
+                double ifInputIsInt = 0;
+                String[] calculatorOperations = { "A", "S", "M", "D" };
+                String[] yesOrNo = { "Y", "N" };
+
+                //tell user to enter a valid alphabet when they enter a int or whitespace
```

Comment:
I won't demand that you delete these comments in your first few projects, but bear in mind that for advanced projects they won't be accepted. The reason is, code should be self-explanatory. Only leave comments that will save a reader from having to figure out what's going on. A good way to make your code self-explanatory is naming your variables and methods as clearly as possible.